RuItUnion.FeedbackBot.Data/FeedbackBotContext.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using RuItUnion.FeedbackBot.Data.Models;
+using TgBotFrame.Commands.Authorization.Interfaces;
+using TgBotFrame.Commands.Authorization.Models;
+
+namespace RuItUnion.FeedbackBot.Data;
+
+public class FeedbackBotContext(DbContextOptions<FeedbackBotContext> options) : DbContext(options), IAuthorizationData
```

Comment:
Suggest to use for default option `QueryTrackingBehavior.NoTracking`. It can speed up work with db

For example:
```csharp
    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
    {
        base.OnConfiguring(optionsBuilder);
        optionsBuilder.UseQueryTrackingBehavior(QueryTrackingBehavior.NoTracking);
    }
```

Reply:
In the solution, `AsNoTracking`/`AsTracking` is called explicitly. I would avoid changing the default option.